### ISSUE TITLE
Errors in X-Piratez

### ISSUE THREAD

**Issue by Pirate300** [Xilmi/OpenXcom#36]
I have encountered two bugs playing X-Piratez:

1- AI doens't use "handle", a weapon of this mod. It makes them only approach without attacking.

2- CTD at certain UFO landing missions. Ending turn just after arriving at certain UFO missions causes the game to crash. 
I will attach my savegame so you can see what is causing this problem.
https://www.mediafire.com/file/798llt1dcoznrjt/ctd.sav/

I have confirmed these bugs don't happen in base OpenXcom

**Comment by Xilmi** [comment]
Can you check if the crash still happens in current version?
It was likely the one introduced in 8.4.0 and fixed in 8.4.1.

The issue with the handle is something I'll still have to look at.

**Comment by Xilmi** [comment]
I investigated the handle-issue.

The reason that they don't want to use it is due to the damage-type-modifiers.
For some reason being hit by a handle restores a ton of morale. So much so, that the AI thinks it would be helping their enemy instead of harming them.

My AI uses some ratios for the damage-types to score the different attack-options. What I hadn't considered is that someone would give weapons negative damage of certain types.

I guess a quick fix would be to ignore these and only consider positive damage-values. A more in-depth-fix would be to simulate the actual effect. A unit with full morale obviously wouldn't benefit from being hit for +858 morale (which is the effect of being hit by the handle with the stats that the soldier I tested it with has).
